### PR TITLE
Cleaned-up sources from my PoC

### PR DESCRIPTION
**Pull request by barnstee**
Code size reduction by 50% while maintaining feature parity (except config of telemetry via separate telemetry config file)
Removed all “using static”s
Made as many class members as possible private
Moved command line arguments parser into separate file
Fixed usage of IDisposable
Removed unsupported Docker configurations
Removed unused Singleton pattern
Fixed loading of cert private key (Martin)
Moved all public configuration settings into separate class
Removed hard-coded UA app and security config and re-enabled UA config file
This led to cert bug above, config via command line in IoT Edge context awkward
Moved telemetry-related source files into separate folder
Moved Heartbeat code into a separate class
Combined all publish unpublish code from 3 places into a single class - No more code duplication with varying degrees of supported features
Removed configurable telemetry message format via config file
Removed subscription and session tracking

### CODE REVIEW

File: opcpublisher/Configurations/Opc.Publisher.Config.xml [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+         
+         The SDK combines these with the base address to create as many EndpointDescriptions as required.
+         Protocols such as HTTP only allow one SecurityPolicy per endpoint to addition URLs are constructed
+         using the SecurityPolicy and SecurityMode.
+         
+         The first policy in the list is assigned to base address. -->
+    <SecurityPolicies>
+      <ServerSecurityPolicy>
+        <SecurityMode>SignAndEncrypt_3</SecurityMode>
+        <SecurityPolicyUri>http://opcfoundation.org/UA/SecurityPolicy#Basic256Sha256</SecurityPolicyUri>
```
Comment (mregen) [inline opcpublisher/Configurations/Opc.Publisher.Config.xml:162]: >http://opcfoundation.org/UA/SecurityPolicy#Basic256Sha256 [](start = 27, length = 57)

empty field autofills with non deprecated policies

File: opcpublisher/Configurations/Opc.Publisher.Config.xml [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+    This ensures subscriptions are not set to expire too quickly. The revised lifetime count
+    and keep alive count are calculated with this value. -->
+    <MinSubscriptionLifetime>10000</MinSubscriptionLifetime>
+    <MaxPublishRequestCount>20</MaxPublishRequestCount>
+    <MaxSubscriptionCount>100</MaxSubscriptionCount>
+    <MaxEventQueueSize>10000</MaxEventQueueSize>
+
+    <!-- see https://opcfoundation-onlineapplications.org/profilereporting/ for list of available profiles -->
+    <ServerProfileArray>
+      <ua:String>Standard UA Server Profile</ua:String>
```
Comment (mregen) [inline opcpublisher/Configurations/Opc.Publisher.Config.xml]: >Standard UA Server Profile [](start = 17, length = 26)

[http://opcfoundation.org/UA-Profile/Server/StandardUA2017](http://opcfoundation.org/UA-Profile/Server/StandardUA2017)

File: opcpublisher/Configurations/Opc.Publisher.Config.xml [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+      <ua:String>Data Access Server Facet</ua:String>
```
Comment (mregen) [inline opcpublisher/Configurations/Opc.Publisher.Config.xml]: >>Data Access Server Facet</ua:St [](start = 16, length = 32)

DA and Method are probably better removed

Comment (barnstee) [reply opcpublisher/Configurations/Opc.Publisher.Config.xml]: Done.

File: opcpublisher/Configurations/Opc.Publisher.Config.xml (hunk 2 above)
Comment (barnstee) [reply opcpublisher/Configurations/Opc.Publisher.Config.xml]: Done.

File: opcpublisher/Configurations/Opc.Publisher.Config.xml (hunk 1 above)
Comment (barnstee) [reply opcpublisher/Configurations/Opc.Publisher.Config.xml:162]: ok

**Pull request closed without merging**